### PR TITLE
add missing crc32 to beacon in xtce generator

### PR DESCRIPTION
**Pull request by ryanpdx**
The tailing CRC32 field was missing from the XTCE generator

It is hard-coded as it is a fully generated value and not in the C3's OD

### CODE REVIEW

File: oresat_configs/scripts/gen_xtce.py [hunk 1]
```diff
@@ -144,7 +144,26 @@ def write_xtce(config: OreSatConfig, dir_path: str = ".") -> None:
     tm_meta = ET.SubElement(root, "TelemetryMetaData")
     tm_meta_para = ET.SubElement(tm_meta, "ParameterTypeSet")
 
-    # Hard-code the 128b type for the AX.25 parameter
+    # hard-code the uint32 time type for the crc32
```
Comment (ThirteenFish) [inline oresat_configs/scripts/gen_xtce.py:147]: Can remove the word `time` here. Looks like it was a copy/paste from `unix_time`?

**Review by ThirteenFish — COMMENTED** [review]
There's now 3 uint32 types in the generated xtce, `uint32_type`, `uint32_s_type` and `uint32_B_type`. Would it make sense to either consolidate them or rename this hard-coded type to be something like `uint32_crc_type` or just `crc32_type`?

Other than that this looks good.

**Comment by ryanpdx** [comment]
> There's now 3 uint32 types in the generated xtce, `uint32_type`, `uint32_s_type` and `uint32_B_type`. Would it make sense to either consolidate them or rename this hard-coded type to be something like `uint32_crc_type` or just `crc32_type`?

The type names current follow `[datatype]_type` for types without units or `[datatype]_[unit]_type` for types with units.

The `uint32_type` is already shared by other fields in the beacon, I just hard-coded it to make sure it always exist for the CRC.

File: oresat_configs/scripts/gen_xtce.py (hunk 1 above)
Comment (ryanpdx) [reply oresat_configs/scripts/gen_xtce.py]: Fixed

**Review by ThirteenFish — APPROVED** [review]

**Pull request merged**